llvm/docs/RemoveDIsDebugInfo.md

```diff
@@ -172,6 +172,24 @@ If you are trying to insert at the start of a block, or purposfully skip debug i
 
 `LLVMPositionBuilder` and `LLVMPositionBuilderBefore` are unchanged. They insert before the indicated instruction but after any attached debug records.
 
+`LLVMGetFirstDbgRecord`, `LLVMGetLastDbgRecord`, `LLVMGetNextDbgRecord` and `LLVMGetPreviousDbgRecord` can be used for iterating over debug records in instructions (provided as `LLVMValueRef`).
```

Comment:
Sorry this comment got lost somehow.

Same comment regarding "in" applies here.

Please could you add entries in to the `New functions (no plans to deprecate)` section (15 or so lines above this) in addition to this part.

Otherwise the docs updates look good, thank you

Reply:
Done